hedera-node/hedera-mono-service/src/main/resources/feeSchedules.json

```diff
@@ -4800,6 +4846,52 @@
           ]
         }
       },
+      {
```

Comment:
We should change all the `feeSChedule.json` available in the repo. Expecially ones under `configuration/mainnet` and `configuration/testnet`

Reply:
I will add the feeSchedules.json under `mono/src/test` also.